cmd/generator/add.go

```diff
@@ -81,12 +103,21 @@ var addCmd = &cobra.Command{
 			return errors.Wrap(err, "failed to download plugin signature")
 		}
 
+		labels := []model.Label{}
```

Comment:
nit:
```suggestion
		var labels []model.Label
```

Reply:
I think there is a noticeable nuisance here: `var labels []model.Label` will render to `null` in json, while `labels := []model.Label{}` renders to `[]` in json. I'm slightly favouring the latter as `null` might be confusing to read. 1/5

Reply:
Ah, do we not `omitempty` for this case?

Reply:
No, we don't (https://github.com/mattermost/mattermost-marketplace/blob/master/internal/model/plugin.go#L17). We also don't `omitempty` for any other field. Hence, I did not add it for consistency.